src/images/icons.svg

```diff
@@ -1,142 +1,143 @@
 <svg xmlns="http://www.w3.org/2000/svg">
   <defs>
-    <symbol viewBox="0 0 16 16" id="vjs-icon-play">
-      <path d="M2 1v14l12-7z"></path>
+    <symbol viewBox="0 0 48 48" id="vjs-icon-play">
```

Comment:
These are much more consistent now! Just out of curiosity, how were these regenerated?

Reply:
@wseymour15 thank you for taking the time to review and comment 👍🏽 

To answer your question: 

It was quite a process.

My original idea was to automate everything by starting from the [videojs/font](https://github.com/videojs/font) project, extracting the icons directly from there and passing them to [svg-screact-cli](https://github.com/elrumordelaluz/svg-spreact-cli). Unfortunately, the installation of videojs/font failed, so I had to abandon the idea to avoid wasting too much time.

So I created a separate project, added the [material-design-icons@v3.0.1](https://www.npmjs.com/package/material-design-icons) and `svg-spreact-cli` dependencies. I then retrieved each `material-design-icons` icon used in https://videojs.github.io/font/ one by one and put them in a directory named svg. However, `videojs/font` has a few `custom icons` which I also retrieved and put in the svg directory. I renamed each svg file to match the icon name.

![Screenshot from 2023-08-01 12-26-52](https://github.com/videojs/video.js/assets/34163393/a83aa8eb-9ce3-4f7b-b8ad-aa5f7233e111)

Once this preparatory work was done, I ran the command: `npx svg-spreact ./svg > icons.svg -p 'vjs-icon-'`

It generates a single svg file that looks like:

![Screenshot from 2023-08-01 12-28-04](https://github.com/videojs/video.js/assets/34163393/8d0e1ab3-5d6f-44fb-8f07-530665db7b7c)

Finally, I removed the `use xlink:href` tags, reordered the generated symbols to make the git diff a little more digestible, removed the `xmlns` properties, checked that everything was coherent and consistent and congratulated myself on a job well done. 😅

Reply:
@wseymour15 I've created a small [project](https://github.com/amtins/svg-sprite-icons/tree/main) to test the automation of svg generation with [svg-sprite](https://github.com/svg-sprite/svg-sprite) and it works quite well and is quite simple. The demo [page](https://amtins.github.io/svg-sprite-icons/images/sprite.symbol.html).